contributing/code/standards.rst

```diff
@@ -143,8 +143,9 @@ Structure
 * Add a blank line before ``return`` statements, unless the return is alone
   inside a statement-group (like an ``if`` statement);
 
-* Use just ``return;`` instead of ``return null;`` when a function must return
-  void early;
+* Use ``return null;`` when a function explicitly returns ``null`` values and
+  use ``return;`` when the function returns ``void`` values (available from
+  PHP 7.1);
```

Comment:
I indeed think the part between parenthesis can be removed as it doesn't add anything really usefull (and removing it avoids confusion)